kuma/settings/common.py

```diff
@@ -367,6 +367,9 @@ def enable_candidate_languages():
 
 SERVE_MEDIA = False
 
+SERVE_LEGACY = config('SERVE_LEGACY', default=False)
```

Comment:
Please add a short comment for this section, like "serve diagrams, presentations, and samples from 2005-2012".

``SERVE_LEGACY`` should include ``cast=bool``.
* Add